### ISSUE TITLE
use `NoneType` as the type of `None`

### ISSUE THREAD

**Issue by KotlinIsland** [KotlinIsland/basedmypy#343]
would fix a lot of issues

**Comment by Zeckie** [comment]
So is that it would treat `None` and `NoneType` as being the same? Otherwise, wouldn't it fail if you used any stubs written for mypy?

**Comment by KotlinIsland** [comment]
> treat `None` and `NoneType` as being the same?

No, there's issues with how mypy handles `None` and `NoneType` that leave a lot to be desired, if your offering to create a PR I could fill you in on what needs to be changed and why.
